### ISSUE TITLE
NIMH Applet Displays Twice in Admin Dashboard

### ISSUE THREAD

**Issue by WorldImpex** [ChildMindInstitute/mindlogger-app#706]
Expected Behavior: After adding NIMH applet to dashboard, user only sees one icon for applet. 
Actual Behavior: After user adds NIMH applet, the dashboard displays two icons. 

![image.png](https://images.zenhubusercontent.com/5e1336f1b24025a7b2654475/7ee6a342-3c92-49d8-a4e4-4de29ff382b8)

**Comment by shnizzedy** [comment]
@WorldImpex, did you already have an applet for that protocol when you added one? If so, this is expected behavior: you'd have two applets with the same protocol but different users and schedules. https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/609 will help clarify this view once resolved.

**Comment by WorldImpex** [comment]
@shnizzedy This is actually @devbtech's account and he was adding applets for the first time and I noticed the same thing that happened to me. He loaded NIMH on once and two icons show up.

**Comment by shnizzedy** [comment]
The quickest thing to check would be to call `GET /user/applets?role=coordinator&ids_only=true` and see if any applet IDs are duplicated.

@devbtech, since there's an applet for NIMH content on each prod & dev, and I'm pretty sure I invited you to manage or coordinate each of those, if you create an applet with that same protocol, that would bring your total of applets with that protocol to 2.

There was an issue where [all applets for which a user has a "coordinator" role were showing up in duplicate](https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/610), but I believe [that issue was resolved](https://github.com/ChildMindInstitute/mindlogger-app-backend/commit/5f469c12b0cfe87d042614e02d501e64ed486217). Since the screenshot shows 2 of only one of the protocols and 1 of each of the others, I suspect these are different applets with the same protocol.

Resolving https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/609 would probably help clear up this issue. As a short-term fix, what about adding the applet ID or number of users to the cards so it's clearer which is which when someone has multiple applets with the same protocol?
